### PR TITLE
experimentally: allow small hight lifts

### PR DESCRIPTION
**Pull request by fieker**
julia> k, a = wildanger_field(3, 13)
(Number field of degree 3 over QQ, _$)

julia> c, mc = Hecke.unramified_completion(k, prime_ideals_over(maximal_order(k), 17)[1]) (Unramified extension of 17-adic numbers of degree 1, Map: k -> c)

julia> preimage(mc, mc(a))
2639044339939425523644123345123996041787685738888204449072451948418018872324117

julia> preimage(mc, mc(a); small_lift = true, integral = true) _$

julia> preimage(mc, mc(a^3//5-2); small_lift = true) 13//5*_$^2 - 13//5*_$ + 3//5

(and the same for plain completion)

needs tests and polish, but Simon wanted to play

### CODE REVIEW

**Comment by thofma** [comment]
Can someone supply an example? Then we can get a test and get this in

**Comment by simonbrandhorst** [comment]
This seems to work for unramified extensions, but for ramified ones not. 
And there is another bug for `setprecision!` documented as a broken test @joschmitt

File: test/LocalField/Completions.jl [hunk 1]
```diff
@@ -37,6 +46,10 @@ end
   @test !iszero(bb)
   @test valuation(bb) == 0
   @test precision(bb) >= 20
+
+  @test_broken bb == preimage(mC, bb; small_lift=true)
```
Comment (simonbrandhorst) [inline test/LocalField/Completions.jl:50]:
```suggestion
  @test bb == preimage(mC, bb; small_lift=true)
```

File: test/LocalField/Completions.jl [hunk 2]
```diff
@@ -37,6 +46,10 @@ end
   @test !iszero(bb)
   @test valuation(bb) == 0
   @test precision(bb) >= 20
+
+  @test bb == preimage(mC, bb; small_lift=true)
+  setprecision!(mC, 21)
+  @test_broken setprecision!(mC, 20)
```
Comment (simonbrandhorst) [inline test/LocalField/Completions.jl:52]:
```suggestion
  setprecision!(mC, 61)
  @test b == preimage(mC, mC(b); small_lift=true)
  @test_broken setprecision!(mC, 20)
```

**Pull request merged**